### ISSUE TITLE
Can't compile against 19.07

### ISSUE THREAD

**Issue by navratk** [ppetr/openwrt-miredo#5]
Hallo. I've got compile error:
<pre>
miredo.c: In function 'miredo':
miredo.c:147:3: error: format not a string literal and no format arguments [-Werror=format-security]
   syslog (LOG_INFO, _("Starting..."));
   ^~~~~~
</pre>
Can be solbed by:
<pre>
PKG_CHECK_FORMAT_SECURITY:=0
</pre> ??
Thx.

**Comment by ppetr** [comment]
Yes, that looks like a pretty harmless warning. Please update here if you encounter other compilation problem, or if you manage to compile it successfully. Thanks.
